committeeoversightapp/management/commands/import_data.py

```diff
@@ -6,13 +6,16 @@
 from django.core.management.base import BaseCommand, CommandError
 from django.core.exceptions import ObjectDoesNotExist, MultipleObjectsReturned
 from django.conf import settings
-from django.db import IntegrityError
+from django.db import transaction, connection
 from django.db.models import Q
 
+
 from opencivicdata.core.models import Organization, OrganizationName
 from opencivicdata.legislative.models import Event, EventSource, EventParticipant
 from committeeoversightapp.models import HearingCategory, Committee
 
+ScrapedEvents = Event.objects.exclude(sources__note='spreadsheet')
```

Comment:
At this point, is it possible that we'd be running the import script once Lugar has entered hearings using the web form? I hope not, but if so I think we'd need to check that imported hearings aren't in that queryset as well.

Reply:
i don't think I understand this. Could you expand.

Reply:
Here's how the web form saves sources:

`source = EventSource(event=event, note="web form")`

So, instead of ScrapedEvents you could have something like ExistingEvents that also includes hearings with `(sources__note='web form')`

Reply:
Actually never mind, sorry! I misread the query and didn't realize it was just excluding spreadsheet imports. Still, the name ScrapedEvents would be misleading if the DB has anything entered from the web form and I'd rename this to ExistingEvents or something similar.